staking/apr/compute.go

```diff
@@ -87,25 +94,25 @@ func ComputeForValidator(
 	)
 
 	if err != nil {
-		return &zero, err
+		return &zero, ErrInsufficientEpoch
 	}
 
 	blockNumAtOneEpochAgo := shard.Schedule.EpochLastBlock(oneEpochAgo.Uint64())
-
 	headerOneEpochAgo := bc.GetHeaderByNumber(blockNumAtOneEpochAgo)
+
 	if block.Header() == nil || headerOneEpochAgo == nil {
```

Comment:
pretty sure have invariant to assume that `.Header() != nil` by caller, can prob drop.